data/json/npcs/missiondef.json

```diff
@@ -513,7 +513,7 @@
     "followup": "MISSION_KILL_HORDE_MASTER",
     "dialogue": {
       "describe": "You seem to know this new world better than most...",
-      "offer": "You're kitted out better than most... would you be interested in making this world a little better for the rest of us?  The towns have enough supplies for us survivors to start securing a foothold but we don't have anyone with the skills and equipment to thin the masses of undead.  I'll lend you a hand to the best of my ability but you really showed promise taking out that other beast.  You, I, and a 100 regular zombies laid to rest, what do you say?",
+      "offer": "You're kitted out better than most... would you be interested in making this world a little better for the rest of us?  The towns have enough supplies for us survivors to start securing a foothold but we don't have anyone with the skills and equipment to thin the masses of undead.  I'll lend you a hand to the best of my ability but you really showed promise taking out that other beast.  You, I, and a 100 zombies laid to rest, what do you say?",
```

Comment:
The PR to address this (#23460) isn't merged yet, so for now the correct string is still `100 regular zombies`. By the way, that PR also changes this string, so your change isn't needed.

Reply:
Hahaha, I really thought this feature was done already. Whoops. Will remove it ASAP.